server/chat-plugins/blackjack.js

```diff
@@ -0,0 +1,643 @@
+/**
+ * Blackjack Game
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This allows users to play the classic blackjack card game.
+ * Credits: jd, panpawn
+ *
+ * @license MIT license
+ */
+'use strict';
+
+class Blackjack extends Rooms.RoomGame {
+	constructor(room, user, target) {
+		super(room);
+		this.room = room;
+
+		this.turnTimeoutMinutes = 1;
+		this.timerTickSeconds = 5;
+
+		this.blackjackNumber = 0;
```

Comment:
nit but why not use `room.gameNumber`

Reply:
I don't think that was apart of the room object when I originally wrote this, I'll update it to use that soon.